build/python/build/libs.py

```diff
@@ -127,7 +147,7 @@
         '--disable-manual',
         '--disable-threaded-resolver', '--disable-verbose', '--disable-sspi',
         '--disable-crypto-auth', '--disable-ntlm-wb', '--disable-tls-srp', '--disable-cookies',
-        '--without-ssl', '--without-gnutls', '--without-nss', '--without-libssh2',
+        '--without-ssl', '--with-wolfssl', '--without-gnutls', '--without-nss', '--without-libssh2',
```

Comment:
This should be a separate commit.

Reply:
This adds ssl authentication. i.e. https by wolfssl while openssl is disabled. It is required for skysight access. Are you sure you want it be a separate commit?

Reply:
Yes, this has nothing to do with Skysight, even though it's a precondition for the Skysight code.

Reply:
I'll create a new commit as the first patch which has "add ssl authentication, https by wolfssl" as commit message.